railties/lib/rails/commands/routes/routes_command.rb

```diff
@@ -0,0 +1,38 @@
+# frozen_string_literal: true
+
+require "rails/command"
+
+module Rails
+  module Command
+    class RoutesCommand < Base # :nodoc:
+      no_commands do
+        def help
+          say "Usage: Print out all defined routes in match order, with names."
+          say ""
+          say "Target specific controller with -c option, or grep routes using -g option"
+          say ""
+        end
+      end
+
+      def perform(*)
+        require_application_and_environment!
+        require "action_dispatch/routing/inspector"
+
+        all_routes = Rails.application.routes.routes
+        inspector = ActionDispatch::Routing::RoutesInspector.new(all_routes)
+
+        routes_filter = nil
+
+        if (controller_search = ARGV.index { |arg| arg == "-c" })
```

Comment:
I think command allow you to define options to not need to build your own option parse for each command. Can you change it to use the options like https://github.com/rails/rails/blob/f7157e48ea233eab56844dba4203f5149d910eaa/railties/lib/rails/commands/server/server_command.rb#L114?

Reply:
Thanks for the review and the tips. I didn't know you can do this. I made the change.